data/json/items/tool_armor.json

```
@@ -1961,37 +1961,112 @@
   {
     "id": "mask_gas",
     "type": "TOOL_ARMOR",
-    "name": { "str": "gas mask" },
     "category": "clothing",
-    "description": "A full gas mask that covers the face and eyes.  Provides excellent protection from smoke, tear gas, and other contaminants.  It must be prepared before use.",
-    "flags": [ "SLEEP_IGNORE" ],
-    "weight": "897 g",
-    "volume": "1 L",
-    "price": "49 USD",
+    "name": { "str": "gas mask" },
+    "description": "A full-face, industrial-grade respirator made from a solid black silicon and butyl rubber blend, providing decent user comfort in addition to a clear field of view and a secure fit thanks to the mask’s single-element vizor and 6-point head harness.  Provided that a filter canister’s been screwed into the right-side filter port and the respirator’s been prepared, it’ll do an admirable job at preventing smoke, gas, and contaminants derived from chemical and lab work from taking up residence in your organs.",
```

visor is a more common spelling.
```suggestion
    "description": "A full-face, industrial-grade respirator made from a solid black silicon and butyl rubber blend, providing decent user comfort in addition to a clear field of view and a secure fit thanks to the mask’s single-element visor and 6-point head harness.  Provided that a filter canister’s been screwed into the right-side filter port and the respirator’s been prepared, it’ll do an admirable job at preventing smoke, gas, and contaminants derived from chemical and lab work from taking up residence in your organs.",
```